app/views/drawings/_drawing.html.haml

```diff
@@ -55,4 +55,14 @@
             .drawing-row
               %strong Context / Story:
               = drawing.story
-            
+            %br
+            .drawing-row
+              - case drawing.image_consent
+              - when nil
+                %span.fa.fa-question
```

Comment:
As @CathMollie found this `?` confusing, I could change this so that if this field has not been answered yet, I just don't show the consent line at all?

![image](https://cloud.githubusercontent.com/assets/4599695/20329698/d4ba2076-ab91-11e6-826c-8ceb0cd6da6e.png)